glib/gvalue.go

```diff
@@ -235,7 +235,7 @@ func gValue(v interface{}) (gvalue *Value, err error) {
 		return val, nil
 
 	case *Object:
-		val, err := ValueInit(TYPE_OBJECT)
+		val, err := ValueInit(e.TypeFromInstance())
```

Comment:
This looks wrong, if you need this because of the socket, then implement `glib.ValueTransformer` for it. This method gets called by the gValue constructor and should set the correct types. For reference look at https://github.com/go-gst/go-gst/blob/b9552df57e7909f0f3ada3117b00aeef47f64a81/gst/gst_element.go#L632

Reply:
problem in case when i get + set socket property - it is now working, but should. I think this problem not only for sockets

Reply:
fixed